packages/terra-hookshot/tests/nightwatch/components/HookshotContentAttachmentBR.jsx

```diff
@@ -5,7 +5,7 @@ class HookshotExample extends React.Component {
   constructor(props) {
```

Comment:
Sooo these test files were named with the first letters of the content attachments. BR for 'bottom right'. Should those change?

Reply:
Hrmmm.... you are correct....
![](https://media.giphy.com/media/l0MYSCYtvpbUqmG8U/giphy.gif)